README.md

```diff
@@ -61,9 +73,46 @@ Publicly available MBTiles charts can be found from:
 
 Plugin adds support for `/resources/charts` endpoints described in [Signal K specification](http://signalk.org/specification/1.0.0/doc/otherBranches.html#resourcescharts):
 
-- `GET /signalk/v1/api/resources/charts/` returns metadata for all available charts
-- `GET /signalk/v1/api/resources/charts/${identifier}/` returns metadata for selected chart
-- `GET /signalk/v1/api/resources/charts/${identifier}/${z}/${x}/${y}` returns a single tile for selected offline chart. As charts-plugin isn't proxy, online charts is not available via this request. You should look the metadata to find proper request.
+- Return metadata for all available charts
+
+```bash
+# v1 API
+GET /signalk/v1/api/resources/charts/` 
+
+# v2 API
+GET /signalk/v2/api/resources/charts/` 
+```
+
+- Return metadata for selected chart
+
+```bash
+# v1 API
+GET /signalk/v1/api/resources/charts/${identifier}` 
+
+# v2 API
+GET /signalk/v2/api/resources/charts/${identifier}` 
+```
+
+#### Chart Tiles
+Chart tiles are retrieved using the url defined in the chart metadata.
+
+For chart files placed in the path(s) defined in the plugin configuration, the url will be:
+
+```bash
+/chart-tiles/${identifier}/${z}/${x}/${y}
+```
+
+#### Mapbox Styles
+
+For Mapbox Styles JSON files the url returned in the metadata will be:
+
+```bash
+/chart-styles/${mapboxstyle.json}
+
+# when access token is defined
+/chart-styles/${mapboxstyle.json}?access_token=${token}
```

Comment:
This is local url, right? So why is there an access token needed?

Reply:
The Mapbox style json file is served locally but may reference resources that are remote which require an access token...
_Example: satellite-v9.jon_
```javascript
{
  "version": 8,
  "name": "Mapbox Satellite",
  "metadata": {
    "mapbox:autocomposite": true,
    "mapbox:type": "default"
  },
  "sources": {
    "mapbox": {
      "type": "raster",
      "url": "mapbox://mapbox.satellite",    // remote resource
      "tileSize": 256
    }
  },
...
```